sdk/iothub/Microsoft.Azure.Management.IotHub/src/Generated/IotHubResourceOperations.cs

```diff
@@ -292,19 +292,19 @@ public async Task<AzureOperationResponse<IotHubDescription>> CreateOrUpdateWithH
         /// <param name='resourceName'>
         /// Name of iot hub to update.
         /// </param>
-        /// <param name='iotHubTags'>
-        /// Updated tag information to set into the iot hub instance.
+        /// <param name='tags'>
+        /// Resource tags
         /// </param>
         /// <param name='customHeaders'>
         /// The headers that will be added to request.
         /// </param>
         /// <param name='cancellationToken'>
         /// The cancellation token.
         /// </param>
-        public async Task<AzureOperationResponse<IotHubDescription>> UpdateWithHttpMessagesAsync(string resourceGroupName, string resourceName, TagsResource iotHubTags, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<AzureOperationResponse<IotHubDescription>> UpdateWithHttpMessagesAsync(string resourceGroupName, string resourceName, IDictionary<string, string> tags = default(IDictionary<string, string>), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
Another breaking change

Note all of these are coming from swagger or autorest. We don't influence how it is generated in a way that causes these.